Generator/Generator/BuiltinGen.swift

```diff
@@ -813,6 +812,14 @@ func generateBuiltinClasses (values: [JGodotBuiltinClass], outputDir: String?) a
             generateBuiltinMethods(p, bc, bc.methods ?? [], typeName, typeEnum, isStruct: kind == .isStruct)
             generateBuiltinOperators (p, bc, typeName: typeName)
             generateBuiltinConstants (p, bc, typeName: typeName)
+                        
+            p("public func toVariant() -> Variant") {
+                p("Variant(self)")
+            }
+            
+            p("public static func fromVariant(_ variant: Variant) -> Self?") {
```

Comment:
Similar for this one, we should generate:

/// Attempts to turn the Variant into this type.

Ideally, if we have the name of the type, we would use that name here and replace it in the docs.